pkg/util/size/bytes.go

```diff
@@ -68,3 +68,31 @@ func Parse(sizeStr string) (ByteCount, error) {
 
 	return ByteCount(size), nil
 }
+
+// MarshalYAML() serializes a ByteCount into YAML. To maintain a canonical representation
+// of the value and to preserve compatibility with older parsers, the byte count will
+// always be serialized as a plain integer without a suffix.
+func (b ByteCount) MarshalYAML() (interface{}, error) {
+	return uint64(b), nil
+}
+
+// UnmarshalYAML unserializes a YAML representation of the ByteCount.
+func (b *ByteCount) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var intVal uint64
+	err := unmarshal(&intVal)
+	if err == nil {
+		*b = ByteCount(intVal)
```

Comment:
Ignore me, yeah there should be a `return` here. The YAML -> string unmarshaling of an integer value passes, which is why tests aren't erring.